_config.yml

```diff
@@ -26,6 +26,8 @@ menu:
     url:               /news/
   - title:             Discussion
     external_url:      https://github.com/hpc-social/hpc-social.github.io/discussions
+  - title:             Join Us On Mastodon
```

Comment:
I think this gets the same message across:
```suggestion
  - title:             Join Mastodon
```
And it matches the menu much better - doesn't look bad.

![image](https://user-images.githubusercontent.com/814322/204062991-6fc33a38-6aaa-4fc9-a6ae-c76cef70c962.png)

Reply:
Hmm, the context of the comments I got was specifically wanting to join our instance rather than just join Mastodon. For now I'd like to keep the wording as I put it there if that's all right with you.

Reply:
Right, but it's a link from the hpc.social so the implication is "Join our Mastodon"

I'd rather not add rushed changes to the site that are wrong and look kind of messy / bad.

Reply:
Hmm, okay but how about "Join mast.hpc.social"? It has to be clear they are joining through our instance. 

I'l leave it to your judgment but thought the original wording was clear. Whatever you decide is okay.

Reply:
I don't like how that looks either, but you can feel free to merge regardless. I don't have to like everything.

Reply:
I have thought about this more and think you are right about the link text needing to be simple, even just "Mastodon" as you originally suggested. What I can't figure out is whether that should link to our project page for the Mastodon server with all the other explanatory links, which would seem like a good thing to do but then why should one project get more priority over the others in home page, or leave it as it is now with this PR as a quick link to the "About" page that has a "Create Account" link as well as a "Sign In" link. (I don't think that linking directory to the "Create Account" link is the right thing to do as that page is very simple and doesn't have the context that the "About" link provides.)

All this to say that we should think about this some more and take some time to arrive at a good solution and I'm open to other approaches with the above thoughts in mind.

Reply:
> why should one project get more priority over the others in home page,

The home page typically should reflect what people are looking for immediately when they join the site! I think it's OK that Mastodon gets put up and center because a lot of people are going to go to the site explicitly looking for it. They wouldn't know to click on "Projects" first, and even if they did, this saves them a click.

I think what would be a nice interaction is:

- Home page of site (and front page) "Mastodon"
- Changes on Mastodon project page:
  - Masotdon page has current automation to add markdown removed, just links to repo
  - "Join us on Mastodon" as a big blue (btn btn-info or btn-primary) button.

Reply:
This sounds okay to me!